src/OmniSharp.Host/CompositionHostBuilder.cs

```diff
@@ -130,7 +135,20 @@ public static IServiceProvider CreateDefaultServiceProvider(IOmniSharpEnvironmen
             services.Configure<OmniSharpOptions>(configuration);
             services.AddSingleton(configuration);
 
-            services.AddLogging();
```

Comment:
The code below is how logging is supposed to be configured these days. Filtering is a first-class feature for the logging APIs now.